cg/meta/workflow/rnafusion.py

```diff
@@ -64,6 +64,19 @@ def get_deliverables_template_content() -> list[dict]:
             file_path=resources.RNAFUSION_BUNDLE_FILENAMES_PATH,
         )
 
+    @property
+    def get_nextflow_config_content(self) -> str:
+        """Return nextflow config content."""
+        return """process {
+    withName:'MULTIQC' {
+        memory = { 1.GB * task.attempt }
+        time   = { 4.h  * task.attempt }
+        cpus = 2
+        ext.args = ' --data-format json '
+    }
+}
```

Comment:
yes, at least taxprofiler

Reply:
Maybe it's worth making it general and moving it to nf_analysis.py so we don't have to change this twice

Reply:
it's not tested yet for taxprofiler, so I'm not sure if the config will be exactly the same. It will use a config, but if the config is the same, I'm not 100% sure yet. Until then I would rather not add it to a common place since this is quite specific.

Reply:
Sounds good 👍